test/schedule.test.ts

```diff
@@ -97,14 +96,13 @@ describe('Schedule', () => {
 			expect(res.body.data).to.containSubset({
 				hasURL: false,
 				schedule: {
-					day: 6,
+					day: 'E',
 					special: false,
 					classes: [
-						{ class: { name: 'b block class' } },
-						{ class: { name: 'f block class' } },
-						{ class: { name: 'Collaborative Work' } },
-						{ class: { name: 'Activities' } },
-						{ class: { name: 'Block D' } }
+						{ class: { name: 'g block class' } },
+						{ class: { name: 'e block class' } },
```

Comment:
Are these in the correct order? It should be the same as the previous test.

Reply:
good catch